Add Godot Game Awards post (#978)
godotengine/godot-website: collections/_article/godot-game-awards-stayathomedev.md

---

> title: "Meet the Creator Behind the Godot Game Awards"

Reviewer: ```suggestion
title: "Meet the creator behind the Godot Game Awards"
```

> excerpt: "StayAtHomeDev has been sharing his passion in YouTube for over two years and now he's running the awards."

Reviewer: ```suggestion
excerpt: "StayAtHomeDev has been sharing his passion on YouTube for over two years and now he's running the awards."
```

categories: ["showcase"]
author: Emi
image: /storage/blog/covers/godot-awards-stayathomedev.webp
date: 2024-12-09 15:00:00
---

> If you’ve watched Godot-related content on YouTube, [StayAtHomeDev](https://www.youtube.com/@stayathomedev) needs no introduction. He’s been sharing his passion for game development and the Godot Engine through his channel. As the creator of the the popular “[This Week in Godot](https://www.youtube.com/watch?v=h3QeaLnOATM&list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series, and the Godot Game Awards, he’s helped showcase the amazing work of the Godot community. We had the chance to talk with him about his journey, what inspired the awards, and his efforts to support the community.

Reviewer: ```suggestion
If you’ve watched Godot-related content on YouTube, [StayAtHomeDev](https://www.youtube.com/@stayathomedev) needs no introduction. He’s been sharing his passion for game development and the Godot Engine through his channel. As the creator of the popular “[This Week in Godot](https://www.youtube.com/playlist?list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series, and the Godot Game Awards, he’s helped showcase the amazing work of the Godot community. We had the chance to talk with him about his journey, what inspired the awards, and his efforts to support the community.
```
I think this link is preferable as it links to the playlist itself, rather than to one particular video within it with the playlist shown alongside.

### What inspired you to create the Godot Game Awards, and how long has it been running?

> I started the “[This Week in Godot](https://www.youtube.com/watch?v=h3QeaLnOATM&list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series over two years ago on my channel.  At first it started as a compilation of cool projects or games I found throughout each week.  About 7-8 videos in, I thought it would be cool if people could vote for their favorite each week and the engagement exploded.  People would comment every week and eventually I compiled every winner into a final vote for the “[Best Godot Project of the Year](https://www.youtube.com/watch?v=3_3N_nL9gIM)” in 2023.  This year I wanted to expand that into multiple categories, highlighting certain achievements like art design and music, and give attention to other things like plugins, tutorials, even devlogs.  From that came the Godot Game Awards, a compilation of the best Godot-based games, plugins, and projects as voted by viewers on my channel.

Reviewer: ```suggestion
I started the “[This Week in Godot](https://www.youtube.com/playlist?list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series over two years ago on my channel.  At first it started as a compilation of cool projects or games I found throughout each week.  About 7-8 videos in, I thought it would be cool if people could vote for their favorite each week and the engagement exploded.  People would comment every week and eventually I compiled every winner into a final vote for the “[Best Godot Project of the Year](https://www.youtube.com/watch?v=3_3N_nL9gIM)” in 2023.

This year I wanted to expand that into multiple categories, highlighting certain achievements like art design and music, and give attention to other things like plugins, tutorials, even devlogs.  From that came the Godot Game Awards, a compilation of the best Godot-based games, plugins, and projects as voted by viewers on my channel.
```

### How can people get involved in this year’s edition?
People can get involved from now until December 15th by voting on the 20+ different categories with an official 2024 🗳️ [Godot Game Awards Ballot](https://forms.gle/Q4yYy82neZcKA3Ny6).

The winners will be announced in a premiere video on December 28th at 2:00pm EST on my channel so that everyone can watch together and help support the great work by these developers.

### Tell us about your YouTube channel—what kind of content do you focus on, and what motivated you to start creating videos?

> I started [StayAtHomeDev](https://www.youtube.com/@stayathomedev) over two years ago to share my passion for creating through game development.  At the time, I had a potential job offer to be the Web Coordinator for a professional football team in the US.  But I had just been given a grant by Meta for some VR development work with their then Oculus Quest 2 headset so I used that to launch the channel. It really was an exercise to create independent income so I could stay at home with my kids.  That was a lifestyle choice I knew I wanted to make for them, so I took a shot at Youtube.

Reviewer: ```suggestion
I started [StayAtHomeDev](https://www.youtube.com/@stayathomedev) over two years ago to share my passion for creating through game development.  At the time, I had a potential job offer to be the Web Coordinator for a professional football team in the US.  But I had just been given a grant by Meta for some VR development work with their Oculus Quest 2 headset so I used that to launch the channel. It really was an exercise to create independent income so I could stay at home with my kids.  That was a lifestyle choice I knew I wanted to make for them, so I took a shot at YouTube.
```

> I’m very interested in sharing knowledge so the channel has had an element of teaching with tutorials or game development concept videos.  With the “[This Week in Godot](https://www.youtube.com/watch?v=h3QeaLnOATM&list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series, the channel started to steer more towards enriching the Godot community and showcasing what people were creating with the engine.  Now, it’s a mix of tutorials, QAs about general game development topics, and the newest format, interviews.  I am actually most excited about the interview videos because they really give a chance for viewers to peek into real game dev life with stories and advice from developers actively breaking through the industry or those who are already successful, and of course doing it with the Godot Engine.

Reviewer: ```suggestion
I’m very interested in sharing knowledge so the channel has had an element of teaching with tutorials or game development concept videos.  With the “[This Week in Godot](https://www.youtube.com/playlist?list=PLEHvj4yeNfeHArSU6U2a715ssJYYCnKCg)” series, the channel started to steer more towards enriching the Godot community and showcasing what people were creating with the engine.  Now, it’s a mix of tutorials, QAs about general game development topics, and the newest format, interviews.  I am actually most excited about the interview videos because they really give a chance for viewers to peek into real game dev life with stories and advice from developers actively breaking through the industry or those who are already successful, and of course doing it with the Godot Engine.
```

Actually, by this point maybe we don't need to keep linking the series, so maybe just this instead:

```suggestion
I’m very interested in sharing knowledge so the channel has had an element of teaching with tutorials or game development concept videos.  With the “This Week in Godot” series, the channel started to steer more towards enriching the Godot community and showcasing what people were creating with the engine.  Now, it’s a mix of tutorials, QAs about general game development topics, and the newest format, interviews.  I am actually most excited about the interview videos because they really give a chance for viewers to peek into real game dev life with stories and advice from developers actively breaking through the industry or those who are already successful, and of course doing it with the Godot Engine.
```

<iframe width="560" height="315" src="https://www.youtube.com/embed/8nsJw89FG_A" frameborder="0" allowfullscreen style="width: 100%; aspect-ratio: 16 / 9; height: auto;"></iframe>

### How did you discover Godot? When did you start using it? Do you have prior experience with other game engines?
If you look at the earliest videos on the channel, you’ll notice they were all focused on the Unreal Engine.  I actually worked within Unreal first.  Long ago…like long ago…I started making little games with an engine called 3D GameStudio.

> When I started the channel, I knew of Godot (which I thought was called GoDot) but never tried it.  After spending more time on Youtube and lurking around other game dev content, Godot kept popping up, so I took it for a test drive.  I ended up making a video about it which went “viral” for a small channel while my first kid was being born.  There was clearly a lot of interest in the engine so I kept playing around with it and fell in love.  Not only with how streamlined it felt but also because it was open source and had such a strong sense of community and pride.  I decided I wanted to help support that with the channel.

Reviewer: ```suggestion
When I started the channel, I knew of Godot (which I thought was called GoDot) but never tried it.  After spending more time on YouTube and lurking around other game dev content, Godot kept popping up, so I took it for a test drive.  I ended up making a video about it which went “viral” for a small channel while my first kid was being born.  There was clearly a lot of interest in the engine so I kept playing around with it and fell in love.  Not only with how streamlined it felt but also because it was open source and had such a strong sense of community and pride.  I decided I wanted to help support that with the channel.
```

### What advice would you give to someone starting out as a game developer or content creator in the Godot community?
I would say the WORST thing you can do is stop yourself before you even try.  Game development and content creation are both difficult.  They are also largely self-motivated so telling yourself you can’t do them means you likely won’t.

The BEST thing you can do is just try.  If you are interested in making games, download the engine and try.  You are going to hit walls and fail at a number of things but that’s part of the learning process.  The information and tools are out there, you just need to start the climb.

In more applicable terms…start small, remake pong or level 1 in Mario Bros.  If something seems too difficult to understand, break it into smaller pieces.  Don’t try to make an entire 2D platformer controller, make it move left and right first.  And don’t think you’re going to sell your first game and make tons of money.  You’ll put too much pressure on yourself and it won’t be fun anymore, you’ll get frustrated, and you’ll quit.

> For content creation, it’s a marathon so the most important thing is don’t worry about the views.  Make content that you want to make.  If a video gets more views than normal, analyze why, and do it again with a small change.  Youtube is sort of a constantly moving target and you won’t get a good idea of your style, your interest, or what people want to watch for a while.  And even then, those things can always change.  Consistency is the important variable…keep doing it and consistently try to get better.  Better sound, better speaking, better editing, better pacing, better thumbnails, better topics, better storytelling.  There is always something you can improve.  If you create videos consistently and improve with every video, you’ll get to where you want to go and build a ton of new skills along the way.

Reviewer: ```suggestion
For content creation, it’s a marathon so the most important thing is don’t worry about the views.  Make content that you want to make.  If a video gets more views than normal, analyze why, and do it again with a small change.  YouTube is sort of a constantly moving target and you won’t get a good idea of your style, your interest, or what people want to watch for a while.  And even then, those things can always change.  Consistency is the important variable…keep doing it and consistently try to get better.  Better sound, better speaking, better editing, better pacing, better thumbnails, better topics, better storytelling.  There is always something you can improve.  If you create videos consistently and improve with every video, you’ll get to where you want to go and build a ton of new skills along the way.
```

<iframe width="560" height="315" src="https://www.youtube.com/embed/C-sJarU469Y" frameborder="0" allowfullscreen style="width: 100%; aspect-ratio: 16 / 9; height: auto;"></iframe>

### Do you plan to continue organizing the Godot Game Awards in the future? 

> Absolutely!  There have already been nearly 1000 ballots submitted already, which is crazy to me.  People comment how they enjoy finding new Godot projects every week with the weekly video series and tell me that they feel inspired to make something themselves.  That is the best thing I could hope for, that people feel inspired to create.  I’m also a music therapist so I have seen the effect creativity and expression can have on a person’s well-being.  To me, games are an art form and within that lies the ability to create powerful feelings in other people through interactive play.  If one person continues to be inspired by the games showcased in the awards, then I’ll keep making the videos.

Reviewer: ```suggestion
Absolutely!  There have already been nearly 1000 ballots submitted, which is crazy to me.  People comment how they enjoy finding new Godot projects every week with the weekly video series and tell me that they feel inspired to make something themselves.  That is the best thing I could hope for, that people feel inspired to create.  I’m also a music therapist so I have seen the effect creativity and expression can have on a person’s well-being.  To me, games are an art form and within that lies the ability to create powerful feelings in other people through interactive play.  If one person continues to be inspired by the games showcased in the awards, then I’ll keep making the videos.
```

<hr>

<span style="font-size: 28px; font-family: var(--header-font-family); font-weight: 800;">🗳️ [Vote now!](https://forms.gle/Q4yYy82neZcKA3Ny6) 👈<span>

<hr>

> If you want to support the work [StayAtHomeDev](https://www.youtube.com/@stayathomedev) you can visit his [Patreon](https://www.patreon.com/c/StayAtHomeDev_), subscribe to [his channel](https://www.youtube.com/@stayathomedev), or vote for your favorite Godot projects each week.

Reviewer: ```suggestion
If you want to support StayAtHomeDev you can visit his [Patreon](https://www.patreon.com/StayAtHomeDev_), subscribe to [his channel](https://www.youtube.com/@stayathomedev), or vote for your favorite Godot projects each week.
```

> Good luck to all the nominees, and thank you [StayAtHomeDev](https://www.youtube.com/@stayathomedev) for all your work organizing the awards!

Reviewer: ```suggestion
Good luck to all the nominees, and thank you StayAtHomeDev for all your work organizing the awards!
```

### What advice would you give to someone starting out as a game developer or content creator in the Godot community?
I would say the WORST thing you can do is stop yourself before you even try.  Game development and content creation are both difficult.  They are also largely self-motivated so telling yourself you can’t do them means you likely won’t.

The BEST thing you can do is just try.  If you are interested in making games, download the engine and try.  You are going to hit walls and fail at a number of things but that’s part of the learning process.  The information and tools are out there, you just need to start the climb.

> In more applicable terms…start small, remake pong or level 1 in Mario Bros.  If something seems too difficult to understand, break it into smaller pieces.  Don’t try to make an entire 2D platformer controller, make it move left and right first.  And don’t think you’re going to sell your first game and make tons of money.  You’ll put too much pressure on yourself and it won’t be fun anymore, you’ll get frustrated, and you’ll quit.

Reviewer: ```suggestion
In more applicable terms…start small, remake Pong or level 1 of Mario Bros.  If something seems too difficult to understand, break it into smaller pieces.  Don’t try to make an entire 2D platformer controller, make it move left and right first.  And don’t think you’re going to sell your first game and make tons of money.  You’ll put too much pressure on yourself and it won’t be fun anymore, you’ll get frustrated, and you’ll quit.
```
